test/lib/ansible_test/_data/sslcheck.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+"""Show openssl version."""
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import json
+
+
+def main():
+    """Main program entry point."""
+    # noinspection PyBroadException
+    try:
+        from ssl import OPENSSL_VERSION_INFO
+        version = list(OPENSSL_VERSION_INFO[:3])
+    except Exception:  # pylint: disable=broad-except
+        version = None
+
+    print(json.dumps(dict(
+        version=version,
+    )))
```

Comment:
nit: Literals rather than constructor.
```suggestion
    print(json.dumps({"version": version}))
```

Reply:
The constructor style is used almost exclusively throughout the entire ansible-test code base, so it's being used here for consistency. It likely has minimal performance impact, especially compared to the subprocess overhead.

Reply:
It's more about readability than performance in this case. I tend to prefer literals, but if using constructors make things more consistent, then it's fine to stick with that.